test/lib/mayaUsd/render/mayaToHydra/cpp/testUtils.cpp

```diff
@@ -528,4 +528,9 @@ bool visibility(const HdSceneIndexBasePtr& sceneIndex, const SdfPath& primPath)
     return (handle ? handle->GetTypedValue(0.0f) : true);
 }
 
+bool contains(const PXR_NS::SdfPathVector& paths, const PXR_NS::SdfPath& path)
+{
+    return std::count(paths.begin(), paths.end(), path) > 0;
```

Comment:
Why not using :
std::find(paths.begin(), paths.end(), path) != paths.end())
As count will not stop at the first occurence since std::find will, so I expect it to be faster.

Reply:
Good catch!